src/components/Header/Header.scss

```
@@ -0,0 +1,24 @@
+header{
+    display: flex;
+    justify-content: space-between;
+    padding: 8px;
+
+    div{
+        font-family: 'Noto Sans KR', sans-serif;
+        font-weight: 600;
+        font-size: 30px;
+        margin-left: 50px;
+    }
+    button{
+        width: 90px;
+        height: 40px;
+        border-radius: 30px;
+        background-color: #3A86FF;
```

`@import "/src/styles/common.scss";` 를 추가하여

```suggestion
        background-color: $main-blue;
```
이런 식으로 변수로 사용할 수 있습니다